### ISSUE TITLE
Security artifacts missing in new kustomize-formatted repositories for MQ workstreams

### ISSUE THREAD

**Issue by osowski** [cloud-native-toolkit/multi-tenancy-gitops#105]
In the new kustomize-formatted GitOps repositories, the security artifacts necessary to deploy MQ with security enabled are currently missing. Below is a snapshot of most of what is needed (the screencap is missing `prod` artifacts but those are required to be ported as well).

![Screen Shot 2021-09-03 at 7 59 04 AM](https://user-images.githubusercontent.com/3410195/132019192-82ac0e37-ea34-46cc-9ecc-55e835572014.png)

The corresponding source YAMLs need to be ported over to the proper place in `multi-tenancy-gitops-apps/tree/master/mq/environments/{dev,staging,prod}/certificates` from `/multi-tenancy-gitops/tree/demo-mq-v2/2-services/instances/namespace-{dev,staging,prod}/secrets` for the certificates, as well as the ClusterIssuer and Certificate from `/multi-tenancy-gitops/tree/demo-mq-v2/2-services/instances/namespace-tools`.

References:
- Source: https://github.com/cloud-native-toolkit/multi-tenancy-gitops/tree/demo-mq-v2
- Target: https://github.com/cloud-native-toolkit-demos/multi-tenancy-gitops-apps

**Comment by osowski** [comment]
So far I've made edits in my forked repos that should cover most of this:

- https://github.com/cloud-native-toolkit-demos/multi-tenancy-gitops-mq/compare/ocp47-2021-2...osowski-demos:ocp47-2021-2-PR-certificates
- https://github.com/cloud-native-toolkit/multi-tenancy-gitops-services/compare/ocp47-2021-2...osowski-demos:ocp47-2021-2-PR-certificates
- https://github.com/cloud-native-toolkit-demos/multi-tenancy-gitops-apps/compare/ocp47-2021-2...osowski-demos:ocp47-2021-2-PR-certificates

I want to confirm that this is the expected pattern moving forward and then I can create formal PRs etc.

**Comment by hollisc** [comment]
Reviewed the edits, 

- For the subscription of the Cert Manager, I just installed it on my cluster and it still installed v0.15.3.  I agree and think we should switch over to the RedHat Marketplace operator.

- The clusterissuer , mq-selfsigned-certificate and mq-selfsigned-clusterissuer in the tools folder (https://github.com/osowski-demos/multi-tenancy-gitops-apps/tree/ocp47-2021-2/mq/config/argocd/tools) looks good but at some point, would like to revisit to see if we can get the sync-wave added to the yamls so we can have a kustomization.yaml.  I had tried this previously but encountered an issue where the proper sequence was not followed causing the resources to not be created but wondering if we add in a custom check of the CR will help.  This all to use kustomization.yaml to reduce a few argocd apps.
